app/src/main/kotlin/com/hippo/ehviewer/client/parser/ArchiveParser.kt

```diff
@@ -62,7 +62,7 @@ object ArchiveParser {
         val funds = maybeFunds ?: PATTERN_CURRENT_FUNDS.find(body)!!.groupValues.run {
             val fundsGP = ParserUtils.parseInt(get(1), 0)
             val fundsC = ParserUtils.parseInt(get(2), 0)
-            Funds("%,d".format(fundsGP), "%,d".format(fundsC))
+            Funds(fundsGP, fundsC)
```

Comment:
It's less readable without separators.

Reply:
Well, how can I add separators to RollingNumber?

Reply:
Ok, I will add it

Reply:
Done